masonry_core/src/core/properties.rs

```diff
@@ -71,38 +91,55 @@ impl Properties {
 
 impl PropertiesRef<'_> {
     /// Returns `true` if the widget has a property of type `P`.
```

Comment:
Sounds like we should have a term for this that we use consistently, like "local property" (this is a term in another system so it would be a good choice).